docs/DeveloperGuide.adoc

```diff
@@ -673,6 +673,64 @@ image::ChangeThemeSequenceDiagram.png[align="center"]
 ** Cons: The command implementation would be more complex as we need to interact with `Model` and `Storage` components too.
 
 // end::theme[]
+// tag::sort[]
+
+=== Sorting Contacts
+==== Current Implementation
+This command sorts all the contacts in TeachConnect lexicographically.
+The command calls upon a `sortByNameFilteredPersonList()` method in `Model`, which then calls upon the `sortList()` method in `UniqueContactList`. The `sortList()` method sorts the entire contact list using a comparator.
+Below is the code snippet.
+
+ public void sortList() {
+        Comparator<Person> sortByName = new Comparator<Person>() {
+            @Override
+            public int compare (Person contact1, Person contact2)  {
+                return contact1.getName().fullName.compareToIgnoreCase(contact2.getName().fullName);
+            }
+        };
+        FXCollections.sort(combinedList, sortByName);
+    }
+
+==== Design Considerations
+===== Aspects : Manually sort contacts or to have an automatic sort feature.
```

Comment:
This needs to be a phrase: "eg: Automatic sorting of contacts, implementation of sort contacts